### PR TITLE
Add update and delete methods.

### PR DESCRIPTION
**Pull request by makaroshyna**

### CODE REVIEW

**Review by TonyH277 — APPROVED** [review]
LGTM!

File: src/main/resources/db/changelog/changes/01-create-books-table.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        - createTable:
+            tableName: books
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
```
Comment (karma-o) [inline src/main/resources/db/changelog/changes/01-create-books-table.yaml]: I think we do not need to explicitly set `nullable` constraint as it is a part of `primaryKey`

File: src/main/resources/db/changelog/changes/01-create-books-table.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                  name: price
+                  type: decimal(12, 2)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: description
+                  type: varchar(255)
+              - column:
+                  name: cover_image
+                  type: varchar(255)
```
Comment (karma-o) [inline src/main/resources/db/changelog/changes/01-create-books-table.yaml:41]: I am not sure that 255 would be enough to store image data, but do not really know how much would image need, that can probably be the thing that we need to investigate later on

File: src/main/resources/db/changelog/changes/01-create-books-table.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+              - column:
+                  name: is_deleted
+                  type: boolean
+                  defaultValueBoolean: false
+                  constraints:
+                    nullable: false
```
Comment (karma-o) [inline src/main/resources/db/changelog/changes/01-create-books-table.yaml]:
```suggestion
                    nullable: false

```

File: pom.xml [hunk 4]
```diff
@@ -56,6 +56,15 @@
...
         </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-maven-plugin</artifactId>
+            <version>${liquibase.version}</version>
+        </dependency>
```
Comment (karma-o) [inline pom.xml:67]: if that's a plugin shouldn't it be added in the plugins section of the pom.xml as well? but if it works - it is fine :)

**Review by karma-o — APPROVED** [review]
Some minor comments only, GJ!

**Pull request merged**